selftests/unit/test_runner_queue.py

```diff
@@ -0,0 +1,58 @@
+import argparse
+import shutil
+import tempfile
+import unittest
+
+from multiprocessing import queues
+
+from avocado.core.job import Job
+from avocado.core.result import Result
+from avocado.core.runner import TestRunner
+from avocado.core.tree import TreeNode
+
+
+class TestRunnerQueue(unittest.TestCase):
```

Comment:
@apahim, how about simply:

```
>>> import avocado
>>> q=avocado.core.job.Job()
>>> a=avocado.core.test.DryRunTest(job=q)
>>> a.whiteboard = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
>>> a.get_state()
{'status': None, 'whiteboard': 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa', 'time_start': -1, 'name': '0-DryRunTest', 'job_unique_id': 'f8941aac98cadc2df2f95b2b3ec647e4fa6eea32', 'class_name': 'DryRunTest', 'traceback': None, 'time_elapsed': -1, 'time_end': -1, 'paused': False, 'running': False, 'fail_class': None, 'params': <AvocadoParams *: * ([]),0: /test/* ([])>, 'logdir': '/home/medic/avocado/job-results/job-2018-02-06T12.20-80b9c78/test-results/0-DryRunTest', 'timeout': None, 'fail_reason': None, 'logfile': '/home/medic/avocado/job-results/job-2018-02-06T12.20-80b9c78/test-results/0-DryRunTest/debug.log', 'job_logdir': '/home/medic/avocado/job-results/job-2018-02-06T12.20-f8941aa'}
```

(obviously with tmpdir set in job).

Reply:
I'm testing the queue content from the runner perspective.